binding.cc

```diff
@@ -27,6 +27,7 @@
 #include <node_buffer.h>
 #include <zmq.h>
 #include <assert.h>
+#include <stddef.h>
```

Comment:
I included it because that is where NULL is defined, and I'm using NULL when I test the return value of zmq_socket.